packages/server-extension/src/index.ts

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
+import { PageConfig } from '@jupyterlab/coreutils';
```

Comment:
Looks like `@jupyterlab/coreutils` should then be added to the `dependencies`:

https://github.com/jupyterlite/jupyterlite/blob/21f072b3b57bc67973bf36312b6a4ddffafcfede/packages/server-extension/package.json#L40-L46

Reply:
At some point it would be great to be able to reuse some of the JupyterLab integrity scripts to ensure dependencies are in sync and catch this for us.

Reply:
thanks!